tests/e2e/scripts/init.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Script to install pre-requisites on vsphere node to run e2e test
+
+PATH=$PATH:/usr/local/go/bin/:/usr/local/bin
+echo "Installing Kubectl"
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
+sudo mv kubectl /usr/local/bin/ && \
+chmod a+x /usr/local/bin/kubectl
+
+echo "Installing jq"
+sudo apt -y install jq
+
+echo "Installing Go"
+curl -O https://dl.google.com/go/go1.16.10.linux-amd64.tar.gz
+tar xvf go1.16.10.linux-amd64.tar.gz
+sudo mv go /usr/local
+/usr/local/go/bin/go version
+go version
+
+echo "Installing Virtualbox"
+sudo apt -y update && upgrade
```

Comment:
```suggestion
sudo apt-get -y update && upgrade
```

Considered safer for scripting.